src/components/TabQueryEditor.vue

```diff
@@ -254,6 +262,7 @@
       keymap() {
         const result = {}
         result[this.ctrlOrCmd('l')] = this.selectEditor
+        result[this.ctrlOrCmd('alt+f')] = this.formatSql
```

Comment:
So we probably need to use the common/menu code to provide the context menu functionality (because we'll want to add menu-bar items to do all these things too, and that will also handle the keyboard shortcuts).

I'll need to think about how to do that.
I think we'll just need to import the menu entries as needed, then use them for the context menu, same way we do for the client-side menu. It might be fiddly to do though.

Reply:
I also tried to implement that one, but i guess im to new for that.  
If i tried to add new menu entries (or for testing remove the whole menu object) nothing changed on the menu (also after a restart).

Feel free to move that part of the code :-)